zmon_aws_agent/main.py

```diff
@@ -147,6 +148,7 @@ def main():
         dynamodbs = aws.get_dynamodb_tables(region, infrastructure_account)
         certificates = aws.get_certificates(region, infrastructure_account)
         aws_limits = aws.get_limits(region, infrastructure_account, apps, elbs)
+        sqs = aws.get_sqs_queues(region, infrastructure_account)
```

Comment:
I agree. SQS queues don't change that often. But I would suggest to change the crawler interval for certain types (SQS, probably also RDS and ElastiCache) in a separate pull request, as this seems to change almost the entire main function.